content/en/docs/contribute/style/style-guide.md

```diff
@@ -46,20 +46,24 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 When you refer specifically to interacting with an API object, use [UpperCamelCase](https://en.wikipedia.org/wiki/Camel_case), also known as Pascal Case. When you are generally discussing an API object, use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization).
 
+You may use the word "resource", "API", or "object" to clarify a Kubernetes resource type in a sentence. Note that Kubernetes resource names are capitalized. However, capitalization alone is not a reliable way to identify Kubernetes resource types in a sentence. 
+
+The sentence "However, pods may have multiple volume mounts" describes general concepts, and uses lower case. In contrast, the sentence "The Pod object includes a Volumes field" is talking about API objects, and uses UpperCamelCase. You may also use [Code Style](#code-style) to distinguish API objects. For example, "The `Pod` object includes a `volumes` field."
+
+Capitalization varies in the Kubernetes API. However, API objects are documented in UpperCamelCase. For example, you may see "configMap" in the [API reference](https://kubernetes.io/docs/reference/). It's fine to call that "ConfigMap" instead when writing general documentation. 
```

Comment:
```suggestion
Capitalization varies in the Kubernetes API. However, the `.kind` of an API object is commonly written in UpperCamelCase. For example, you may see `configMap` in some areas of the [API Reference](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/). It's better to call that type of object a `ConfigMap` when writing general documentation.
```

Reply:
Line 53, Again, the sentence is about capitalization/upper camel case. If you feel that this content
clarifies the use of upper camel case in the documentation, you could add it after the first
sentence.

Reply:
I am unsure whether this style information is warranted (line 53).